spajoa.py

```diff
@@ -43,7 +43,7 @@ def run_optimal_737(objective = 'fuel'):
         })
     else:
         substitutions.update({
-           'n_{pax}': [180.],
+           'n_{pax}': 180.,
            'ReqRng': 3000.*units('nmi'),
```

Comment:
@1ozturkbe we need the brackets on Npax for the model to build when not running sweeps. Please check that code works before pushing it.